paper/paper.md

```diff
@@ -124,6 +124,29 @@ The toolkit is written in Python and it has a modular and object-oriented design
 
 ![A simple _Black-it_ calibration script.\label{fig:features}](figures/code_snippet.png){ width=40% }
 
+<!--
+```python
+from black_it import Calibrator
+from black_it.samplers import HaltonSampler, RandomForestSampler
+from black_it.loss_functions import MethodOfMomentsLoss
+# initialise samplers to take 10 samples each
+halton = HaltonSampler(10)
+random_forest = RandomForestSampler(10)
+# initialise loss function
+loss = MethodOfMomentsLoss()
+# initialise calibrator
+cal = Calibrator(
+      samplers=[halton, random_forest],
+      real_data=real_data,
+      model=model,
+      parameters_bounds=bounds,
+      parameters_precision=precisions,
+      loss_function=loss)
+# calibrate for 30 epochs
+params, losses = cal.calibrate(30)
+```
+-->
+
```

Comment:
this is the outcome I get when using the usual Markdown code block, with `python` syntax highlighter:


![Screenshot from 2022-06-28 21-38-05](https://user-images.githubusercontent.com/90861103/176271566-705c1527-3d0e-4611-8ce3-b99815725f51.png)


In particular, note that the highlighting is quite limited, e.g. import keywords are not highlighted.

Reply:
Yes I had obtained something like this when I tried... let's forget about it then